SingularityService/pom.xml

```diff
@@ -14,6 +14,17 @@
     <basepom.shaded.main-class>com.hubspot.singularity.SingularityService</basepom.shaded.main-class>
   </properties>
 
+  <dependencyManagement>
```

Comment:
Hubspot depends on a managed parent pom, and without switching to a provider based methodology, just like MySQL - the PG driver is always "included"